prime-router/docs/playbooks/dependabot-updates.md

```diff
@@ -13,7 +13,10 @@ Below are the steps for how to most safely check and merge in the dependabot PRs
 `@dependabot rebase` to let dependabot rebase the branch for you.  If you use any other method then dependabot will not be able to keep track of the PR.
 1. Verify that the build for the PR is successful.  Note that the unit, integration and smoke tests are run as part of the build.
 1. Read the updated library's changelog and identify and communicate any risks you find.  When in doubt ask! Library changes can affect many parts of the system.
-1. Identify what places in the code the library is used then identify if the unit, integration and/or smoke tests provide enough coverage to verify the update does 
+1. Identify any library version conflicts for the updated library by running the command `./gradlew dependencies | grep <library name>` and verifying that 
```

Comment:
These instructions are also included below. This line can be made simpler.

Reply:
It is good to have the command in the list of steps, here.  Most of the time, it will come out with only one version - done.  However, see my comment below.